CMakeLists.txt

```diff
@@ -28,15 +31,8 @@ include_directories(${MSP_INCLUDE_DIR})
 ################################################################################
 ### libraries
 
-# low-level API
-add_library(msp ${MSP_SOURCE_DIR}/MSP.cpp)
-target_link_libraries(msp ${CMAKE_THREAD_LIBS_INIT})
-
-# printing message content
-add_library(msp_msg_print ${MSP_SOURCE_DIR}/msg_print.cpp)
```

Comment:
Keep this file with streaming operators for messages.

Reply:
that will take some work to migrate everything